docs/developer/developer-setup.md

```diff
@@ -161,23 +161,19 @@ All projects in TruBudget (blockchain, api, frontend, etc) have a docker-compose
 Do **NOT** use other additional ways to set environment variables like inline env-variables or env-variables defined by 'export'. Why? - Because these env-variables will overwrite each which makes it very hard to find mistakes.
 :::
 
-### Docker Environment
-
-There is a folder called `docker-compose` in the root project directory. Running the project from these files is also possible, however check out the [README.md](https://github.com/openkfw/TruBudget/blob/master/docker-compose/README.md) for more details.
-
 ## Development Setup
 
 If you want to start developing on Trubudget, you need to setup the application locally. This guide tells you how to start the blockchain, start the API, load up some test data and start the frontend.
 
-### Dockerized Application
+### Dockerized Application (recommended)
 
-This is the fastes way you can run all services needed for development. Everything is run in one command:
+This is the fastest way you can run **all services** needed for development. Everything is started in one command:
 
-1.  in root directory execute: `sh scripts/development/start-dev.sh --full` or `sh scripts/development/start-dev.sh --slim`
+1.  in root directory execute: `bash scripts/development/start-dev.sh --slim` or `bash scripts/development/start-dev.sh --full`
```

Comment:
Where is step 2?

Reply:
there is only one step thanks to my all-in-one script

Reply:
then remove leading 1

Reply:
no